scripts/build_attestation_agent.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+#
+# Copyright (c) 2022 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+[ -n "${BASH_VERSION:-}" ] && set -o errtrace
+[ -n "${DEBUG:-}" ] && set -o xtrace
+
+source $HOME/.cargo/env
+
+SCRIPT_DIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+AA_DIR=$SCRIPT_DIR/attestation_agent
+
+pushd $SCRIPT_DIR
+git clone --depth 1 "https://github.com/confidential-containers/attestation-agent.git" $AA_DIR
+pushd $AA_DIR
+make KBC=sample_kbc
+install target/release/attestation-agent $SCRIPT_DIR/
+popd
+
+rm -rf $AA_DIR
```

Comment:
This command won't be called if something goes wrong before. I suggest you to create a clean up function to be called on shell's exit. Example:

```bash
cleanup() {
  rm -rf "$AA_DIR"
}
trap cleanup EXIT
```